ansible_collections/arista/avd/roles/eos_designs/docs/tables/wan-cv-pathfinder-regions.md

```diff
@@ -7,6 +7,10 @@
 
     | Variable | Type | Required | Default | Value Restrictions | Description |
     | -------- | ---- | -------- | ------- | ------------------ | ----------- |
+    | [<samp>cv_pathfinder_global_sites</samp>](## "cv_pathfinder_global_sites") | List, items: Dictionary |  |  |  | Define sites that are outside of the CV Pathfinder hierarchy.<br>This is used to arrange pathfinders in the CloudVision topology layout. |
```

Comment:
would be good to add some sentence about this in the how-to/wan.md page in the hierarchy section

Reply:
done https://github.com/aristanetworks/avd/pull/3904/commits/dee286b6dfddf2870292bcc27fe3c3719aee1c7e